### PR TITLE
Add typedoc comments for event manager modules

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Add typedoc comments for event manager (event handler) modules
- Rename: EventConfigType -> BlockchainEventConfig
- Rename: DisconnectCallback -> DisconnectionCallback
- Remove some unused member variables

Related issues:
- https://github.com/ainblockchain/ain-blockchain/issues/1207

### CODE REVIEW

File: src/event-manager/index.ts [hunk 1]
```diff
@@ -49,9 +64,18 @@ export default class EventManager {
     errorCallback?: (error: any) => void,
     filterDeletedEventCallback?: FilterDeletedEventCallback
   ): string;
+  /**
+   * Subscribes to blockchain events.
+   * @param {string} eventTypeStr The event type.
+   * @param {BlockchainEventConfig} config The blockchain event configuraiton.
+   * @param {BlockchainEventCallback} eventCallback The blockchain event callback function.
+   * @param {(error: any) => void} errorCallback The error event callback function.
```
Comment (woomurf) [inline src/event-manager/index.ts:72]: How about define as type? like `BlockchainEventErrorCallback`.

**Review by woomurf — APPROVED** [review]
LGTM with comment. Thanks!

File: src/event-manager/index.ts (hunk 1 above)
Comment (platfowner) [reply src/event-manager/index.ts]: Just done, thanks!

**Comment by platfowner** [comment]
Thanks for the review! @woomurf  Now merging..

**Pull request merged**